src/main/java/uk/gov/hmcts/darts/event/controller/EventsController.java

```diff
@@ -38,13 +43,17 @@ public class EventsController implements EventApi {
     @RequestMapping(
         method = RequestMethod.POST,
         value = "/events",
-        produces = { "application/json" },
-        consumes = { "application/json" }
+        produces = {"application/json"},
+        consumes = {"application/json"}
     )
     @Override
     public ResponseEntity<AddDocumentResponse> eventsPost(
         @Parameter(name = "DartsEvent", description = "") @Valid @RequestBody(required = false) DartsEvent dartsEvent
     ) {
+
+        // send to darts-gateway for DAR PC Notify
+        eventsService.notifyEvent(dartsEvent);
```

Comment:
Depends on scope of this ticket.
this should probably have some sort of "handleEvent" method that looks up the moj_event_type table to figure out what handler to use, and anything that is either "DARTSEventDARStartHandler" or "DARTSEventDARStopHandler", etc, then send the request to DAR PC, but if the scope of the ticket is literally just making the FeignClient to make the request, then should maybe remove this.

Reply:
Yep, the code will need to be iterated on to do this.
We need other stuff to happen first, like `save to db` to be able to work out what `notification type` to send and possibly also what `notification url` too.
This was added here, as I needed a means to be able to add an `integrationTest` to verify the Body looked as expected for the darts-gateway request.